ortb/request_validator.go

```diff
@@ -0,0 +1,172 @@
+package ortb
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+	"github.com/prebid/prebid-server/v2/stored_responses"
+)
+
+type RequestValidator struct {
```

Comment:
Consider changing RequestValidator to an interface with only the ValiateImp method for now. Then rename this "standardRequestValidator" to be responded by `NewRequestValidator`.